omod/src/main/java/org/openmrs/module/aijar/page/controller/MarkPatientDeadPageController.java

```diff
@@ -20,19 +21,18 @@
 /**
  * Created by lubwamasamuel on 15/06/2016.
  */
-public class MarkPatientDeadPageController {
+public class MarkPatientDeadPageController{
     public void controller(UiSessionContext sessionContext, PageModel model) {
     }
 
     public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcrumbOverride",required = false) String breadcrumbOverride, @RequestParam("patientId") String patientId) {
-        PersonService personService = Context.getPersonService();
         PatientService patientService = Context.getPatientService();
 
         String conceptId = Context.getAdministrationService().getGlobalProperty("concept.causeOfDeath");
 
         Patient patient = patientService.getPatientByUuid(patientId);
-        Person person = personService.getPerson(patient.getPatientId());
-        pageModel.put("person", person);
+        pageModel.put("person", patient.getPerson());
```

Comment:
@slubwama Why put a person int he page model when you already have a patient, and can get a person from patient?